test/resources/stubs/testprojekt-ets6-functions.json

```diff
@@ -103,8 +103,20 @@
                             "function_type": "FT-1",
                             "project_uid": 11,
                             "group_addresses": [
-                                "P-05C0-0_GA-1",
-                                "P-05C0-0_GA-2"
+                                {
+                                    "identifier": "P-05C0-0_GF-1",
+                                    "name": "",
+                                    "role": "SwitchOnOff",
+                                    "ref_id": "P-05C0-0_GA-1",
```

Comment:
We should use or add the 3-level group address notation (`1/2/3`) since we use this as keys in the root `group_addresses` key and we will probably have to look up the group address for a given function when it is used somehow.